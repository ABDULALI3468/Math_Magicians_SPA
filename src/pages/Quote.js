/* eslint max-len: ["error", { "code": 1000 }] */
import React from 'react';
import '../components/Navbar/Navbar.css';

const Quote = () => (
  <div className="quote">
    <p>Mathematics is not about numbers,equation, computations,or algorithms : It is about understanding. </p>
  </div>
);

export default Quote;
