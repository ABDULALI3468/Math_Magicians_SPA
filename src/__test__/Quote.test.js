import React from "react";
import renderer from "react-test-renderer";
import Quote from "../pages/Quote";
import "@testing-library/jest-dom";

it('Test the Qutoe page', () => {
  const quote = renderer
    .create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
})