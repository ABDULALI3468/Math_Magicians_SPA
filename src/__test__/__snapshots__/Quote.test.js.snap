// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Qutoe page 1`] = `
<div
  className="quote"
>
  <p>
    Mathematics is not about numbers,equation, computations,or algorithms : It is about understanding. 
  </p>
</div>
`;
